README.md

```diff
@@ -1 +1,3 @@
-# best-repo-ever
\ No newline at end of file
+# best-repo-ever
+Hello Git
```

Comment:
Why this line?

Reply:
This is just to make some changes to the file and see if it gets updated in Git repo